src/sql/azdata.d.ts

```diff
@@ -2209,7 +2209,9 @@ declare module 'azdata' {
 
 	export enum AzureResource {
 		ResourceManagement = 0,
-		Sql = 1
+		Sql = 1,
+		OssRdbms = 2,
+		Graph = 4
```

Comment:
Why are you adding these directly here instead of proposed?

Reply:
It didn't make sense to me to throw this into proposed because it's going to be used by a built-in extension.

Reply:
And it's actually going to be propagating the user settings, etc.

Reply:
Alrighty, if you're doing that why not put all of them in there then? Seems like these are pretty unlikely to ever change anyways right?

Reply:
Yeah the other one was AKV which I'm not directly using. I guess that could come here too.

Reply:
Done!